src/seckrit.py

```diff
@@ -32,7 +46,8 @@ def main(args):
         secret_type = secret_info["type"]
 
         if secret_type == "environment_variable":
-            print("Adding environment variable {} to {}".format(secret_info["destination"], manifest["environment_file"]))
+            print(
```

Comment:
Not sure if this was intentional, but this can remain on one line.

Reply:
Oops! that was Pycharm code reformat. Not sure why it wants things to be ugly 🤷‍♂